.gitignore

```diff
@@ -1,4 +1,5 @@
 node_modules
+.loom
 .sewing-kit
```

Comment:
Can we eventually get rid of this?

Reply:
We can, maybe... I left it so people who have the repo already do not suddenly have a bunch of formally-gitignored files that now show up as added.